### PR TITLE
Add support for skipping loads.

### PR DESCRIPTION
**Pull request by bstein-clever**
This should really only be used in conjunction with job debouncing.

**JIRA**:
https://clever.atlassian.net/browse/DEIP-171

**Overview**:
Hot on the tail of https://github.com/Clever/connections-to-s3/pull/104

Now that it works, we can add debouncing to other services.
s3-to-redshift needs to have an arg from the debouncing, so it knows when a load would be invalid,
so this has to merge before the mongo-to-s3 or record-keeper jobs.

**Testing**:

**Roll Out**:
- [ ] This repo will auto deploy after you merge. To do a manual deploy, use ark start -e production s3-to-redshift or ark start -e production s3-to-redshift-fast

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -290,6 +290,7 @@ type payload struct {
 	StreamStart     string `config:"streamStart"`
 	StreamEnd       string `config:"streamEnd"`
 	TargetTimezone  string `config:"timezone"`
+	SkipLoad        bool   `config:"skip-load"`
```
Comment (jansenclever) [inline main.go:293]: Nit: Should we make this skipLoad to match the case of streamStart/streamEnd? Though I don't what our standard is, seems like it's all over the place

**Review by jansenclever — APPROVED** [review]
LGTM

File: main.go (hunk 1 above)
Comment (jansenclever) [reply main.go:293]: (I don't feel strongly about this)

Comment (jansenclever) [reply main.go:293]: I guess `skip-debounce` also has a dash but we didn't have two word config params in the other repros so I didn't think about it then.

Comment (bstein-clever) [reply main.go]: Chatted offline. Switched it to be camelCase, and consistent across the repos with skipDebounce.

**Pull request merged**